Orange/widgets/data/owsql.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python
```

Comment:
As widgets are mostly used from Canvas, we usually do not include this. For testing, widget can be run with python widget_module.py.